docs/src/routes/docs/components/dropdown.svx

```diff
@@ -101,6 +123,30 @@ Falsy values passed to classes will be disregarded.
 
 The content of the dropdown.
 
+### Related Objects
+
+The following objects are available as named exports in 'attractions/dropdown'.
+
+
+#### HorizontalAlignment
+Dropdown content alignment on horizontal axis.
```

Comment:
I suppose it's not really the content that we're aligning, but the dropdown itself: 
```suggestion
Dropdown alignment along the horizontal axis.
```